src/Components/Book.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { BOOK_DELETED } from '../redux/books/booksSlice';
 
-const Book = ({ title, author }) => (
-  <div className="book-container">
-    <h2>{title}</h2>
-    <p>{author}</p>
-    <button type="submit">delete</button>
-  </div>
-);
+const Book = ({ bookId, title, author }) => {
+  const dispatch = useDispatch();
+
+  const handleDeleteBook = () => {
+    dispatch(BOOK_DELETED({ bookId }));
```

Comment:
- [ ] The same error.

Reply:
Thanks very much for the review @Rustamxon7 . However, sample tutorial on the redux toolkits documentation site [see Quickstart](https://redux-toolkit.js.org/tutorials/quick-start), recommends this approach.

Thanks for the good eyes. 🚀